File: MapGUI3/src/Model/expressions/ConstantExpression.java
package Model.expressions;

import Model.Heap;
import Model.SymbolTable;

public class ConstantExpression extends Expression {

    private int number;

    public ConstantExpression(int number) {
        this.number = number;
    }

    public int evaluate(SymbolTable<String, Integer> symTable, Heap ph) {
        return number;
    }

    public String toString() {
        return Integer.toString(number);
    }

}

File: MapA7/MapA7/Model/Expressions/ConstantExpression.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MapA7.Model.Expressions
{
    public class ConstantExpression:IExpression
    {
        private int number;

        public ConstantExpression(int number) { this.number = number; }

        public int Evaluate(SymbolTable symbolTable)
        {
            return number;
        }

        public String Tostring()
        {
            return number.ToString(); 
        }
    }
}
